### ISSUE TITLE
DOTA TV not working on Major games. 3 days in a row =(

### ISSUE THREAD

**Issue by shispgay** [ValveSoftware/Dota2-Gameplay#2022]
### Description

I Cannot watch 50-75% games from Major cuz of dota tv lag. 
0 viewers count seems like proof of problem on server side.
May you fix it already, please?

### Screenshots

![Screenshot_15](https://user-images.githubusercontent.com/101227334/183263783-856864c4-4a91-41f6-845c-a065d324dbf4.png)

**Comment by Gabenfixfordota2** [comment]
I completely agree with this person and I want valve to see this post and fix Dota 2 as soon as possible

**Comment by Potya** [comment]
Surely, it must be fixed

**Comment by betepookk** [comment]
shisp gay

**Comment by Eastofeden1** [comment]
w8ing when jeff hill wakes up

**Comment by AlexSus** [comment]
Still not working!

**Comment by Swenyly** [comment]
Up to now I have not been able to fix it [fleeing the complex](https://fleeingthecomplex.online)

**Comment by jordanlaris** [comment]
Have any update for it? I think we need to improvement [[death run 3d](url)]([death run 3d](https://deathrun3d.com))

**Comment by jennytrippi** [comment]
I watch to 65% then it stops. The situation is the same as you and I are trying to fix it, I can't do it right now. It's quite difficult like [stickman boost](https://stickmanboost.com)
